assignment-client/src/scripts/EntityScriptServer.cpp

```diff
@@ -209,7 +209,7 @@ void EntityScriptServer::selectAudioFormat(const QString& selectedCodecName) {
 }
 
 void EntityScriptServer::resetEntitiesScriptEngine() {
-    auto engineName = QString("Entities %1").arg(++_entitiesScriptEngineCount);
+    auto engineName = QString("about:Entities %1").arg(++_entitiesScriptEngineCount);
```

Comment:
... prefixing with `about:` avoids any `QUrl` code confusing this kind of non-filename with being a local file.